Migrated to latst hugo-book theme. Added hint shortcode override

ogdenpm opened the pull request:
resolves #1
resolves #2
Updated hugo book theme to latest version and created a hit override to support the old behaviour
Modified the hard coded URL to use a site relative name

Robin-Shipp commented:
Hi Mark 

I can see these and I think I understand them. Thanks, but how do I accept a pull request?

I need to go offline for a couple of hours to do something else, so don’t expect a quick reply.

Robin

Robin Shipp
31 Rosslyn Way
Thornbury, Bristol, BS35 1SG
Email: ***@***.***
Mob: 07747 363380







> On 1 Feb 2025, at 11:38, Mark Ogden ***@***.***> wrote:
> 
> 
> @ogdenpm <https://github.com/ogdenpm> requested your review on: #3 <https://github.com/cccbr/belfryupkeep/pull/3> Migrated to latst hugo-book theme. Added hint shortcode override.
> 
> —
> Reply to this email directly, view it on GitHub <https://github.com/cccbr/belfryupkeep/pull/3#event-16149909911>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AR2FQ3BEJKJJZS3OSHVRPRT2NSW4PAVCNFSM6AAAAABWJGOV2CVHI2DSMVQWIX3LMV45UABCJFZXG5LFIV3GK3TUJZXXI2LGNFRWC5DJN5XDWMJWGE2DSOJQHE4TCMI>.
> You are receiving this because your review was requested.
>

ogdenpm commented:
Robin

You should be able to click on “Pull requests” at the top of the page.

Then on the page it shows you should be able to “Merge pull request”.

If you want you can then test locally, easiest is to clone to your local computer and run “hugo server”

If possible, use the git clone –recursive command as otherwise you will need to run the two commands below from the cloned directory

git submodule init

git submodule update

 

Once you are happy you commit the code

 

Note there is a similar pull request on your other website, unfortunately I couldn’t assign you so you may not have an email about it

 

If you would rather me hold a quick demonstration via teams, then let me know.

 

Mark

 

 

 

From: Robin Shipp ***@***.***> 
Sent: 01 February 2025 11:55
To: cccbr/belfryupkeep ***@***.***>
Cc: Mark Ogden ***@***.***>; Mention ***@***.***>
Subject: Re: [cccbr/belfryupkeep] Migrated to latst hugo-book theme. Added hint shortcode override (PR #3)

 

Hi Mark 

I can see these and I think I understand them. Thanks, but how do I accept a pull request? 

I need to go offline for a couple of hours to do something else, so don’t expect a quick reply. 

Robin 

Robin Shipp 
31 Rosslyn Way 
Thornbury, Bristol, BS35 1SG 
Email: ***@***.*** <mailto:***@***.***>  
Mob: 07747 363380 







> On 1 Feb 2025, at 11:38, Mark Ogden ***@***.*** <mailto:***@***.***> > wrote: 
> 
> 
> @ogdenpm <https://github.com/ogdenpm> requested your review on: #3 <https://github.com/cccbr/belfryupkeep/pull/3> Migrated to latst hugo-book theme. Added hint shortcode override. 
> 
> — 
> Reply to this email directly, view it on GitHub <https://github.com/cccbr/belfryupkeep/pull/3#event-16149909911>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AR2FQ3BEJKJJZS3OSHVRPRT2NSW4PAVCNFSM6AAAAABWJGOV2CVHI2DSMVQWIX3LMV45UABCJFZXG5LFIV3GK3TUJZXXI2LGNFRWC5DJN5XDWMJWGE2DSOJQHE4TCMI>. 
> You are receiving this because your review was requested. 
> 




—
Reply to this email directly, view it on GitHub <https://github.com/cccbr/belfryupkeep/pull/3#issuecomment-2628922490> , or unsubscribe <https://github.com/notifications/unsubscribe-auth/AAZ6DDYPRVPVQOFQBAPFW7T2NSY2TAVCNFSM6AAAAABWJGOV2CVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDMMRYHEZDENBZGA> .
You are receiving this because you were mentioned.  <https://github.com/notifications/beacon/AAZ6DD4XMKEF3G2EJRIQOBT2NSY2TA5CNFSM6AAAAABWJGOV2CWGG33NNVSW45C7OR4XAZNMJFZXG5LFINXW23LFNZ2KUY3PNVWWK3TUL5UWJTU4WIWHU.gif> Message ID: ***@***.*** ***@***.***> >

Robin-Shipp commented:
I think I’ve just done this. Does that mean that Belfry Upkeep is now fixed?

I don’t want to get into cloning. I haven’t done it since I cloned Bryn’s Calling it Round when I first got into this. It occurs to me that quite a lot of the stuff in that is what is causing the problems now!

I really do need to log off for a couple of hours now.

Robin

Robin Shipp
31 Rosslyn Way
Thornbury, Bristol, BS35 1SG
Email: ***@***.***
Mob: 07747 363380







> On 1 Feb 2025, at 12:13, Mark Ogden ***@***.***> wrote:
> 
> 
> Robin 
> 
> You should be able to click on “Pull requests” at the top of the page. 
> 
> Then on the page it shows you should be able to “Merge pull request”. 
> 
> If you want you can then test locally, easiest is to clone to your local computer and run “hugo server” 
> 
> If possible, use the git clone –recursive command as otherwise you will need to run the two commands below from the cloned directory 
> 
> git submodule init 
> 
> git submodule update 
> 
> 
> 
> Once you are happy you commit the code 
> 
> 
> 
> Note there is a similar pull request on your other website, unfortunately I couldn’t assign you so you may not have an email about it 
> 
> 
> 
> If you would rather me hold a quick demonstration via teams, then let me know. 
> 
> 
> 
> Mark 
> 
> 
> 
> 
> 
> 
> 
> From: Robin Shipp ***@***.***> 
> Sent: 01 February 2025 11:55 
> To: cccbr/belfryupkeep ***@***.***> 
> Cc: Mark Ogden ***@***.***>; Mention ***@***.***> 
> Subject: Re: [cccbr/belfryupkeep] Migrated to latst hugo-book theme. Added hint shortcode override (PR #3) 
> 
> 
> 
> Hi Mark 
> 
> I can see these and I think I understand them. Thanks, but how do I accept a pull request? 
> 
> I need to go offline for a couple of hours to do something else, so don’t expect a quick reply. 
> 
> Robin 
> 
> Robin Shipp 
> 31 Rosslyn Way 
> Thornbury, Bristol, BS35 1SG 
> Email: ***@***.*** <mailto:***@***.***> 
> Mob: 07747 363380 
> 
> 
> 
> 
> 
> 
> 
> > On 1 Feb 2025, at 11:38, Mark Ogden ***@***.*** <mailto:***@***.***> > wrote: 
> > 
> > 
> > @ogdenpm <https://github.com/ogdenpm> requested your review on: #3 <https://github.com/cccbr/belfryupkeep/pull/3> Migrated to latst hugo-book theme. Added hint shortcode override. 
> > 
> > — 
> > Reply to this email directly, view it on GitHub <https://github.com/cccbr/belfryupkeep/pull/3#event-16149909911>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AR2FQ3BEJKJJZS3OSHVRPRT2NSW4PAVCNFSM6AAAAABWJGOV2CVHI2DSMVQWIX3LMV45UABCJFZXG5LFIV3GK3TUJZXXI2LGNFRWC5DJN5XDWMJWGE2DSOJQHE4TCMI>. 
> > You are receiving this because your review was requested. 
> > 
> 
> 
> 
> 
> — 
> Reply to this email directly, view it on GitHub <https://github.com/cccbr/belfryupkeep/pull/3#issuecomment-2628922490> , or unsubscribe <https://github.com/notifications/unsubscribe-auth/AAZ6DDYPRVPVQOFQBAPFW7T2NSY2TAVCNFSM6AAAAABWJGOV2CVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDMMRYHEZDENBZGA> . 
> You are receiving this because you were mentioned. <https://github.com/notifications/beacon/AAZ6DD4XMKEF3G2EJRIQOBT2NSY2TA5CNFSM6AAAAABWJGOV2CWGG33NNVSW45C7OR4XAZNMJFZXG5LFINXW23LFNZ2KUY3PNVWWK3TUL5UWJTU4WIWHU.gif> Message ID: ***@***.*** ***@***.***> > 
> 
> —
> Reply to this email directly, view it on GitHub <https://github.com/cccbr/belfryupkeep/pull/3#issuecomment-2628928782>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AR2FQ3FVJWGFX5G6F7WGXXD2NS26NAVCNFSM6AAAAABWJGOV2CVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDMMRYHEZDQNZYGI>.
> You are receiving this because your review was requested.
>
